### DISCUSSION TITLE
How to multiple images with thesame person

### DISCUSSION THREAD

**Discussion by EmeraldTablet** [lllyasviel/Fooocus#1722] [Q&A]
I'd like to generate multiple images of the same character to visualize in different poses, positions, scenarios, etc. But every generation is a new character altogether so just wondering if anyone's figured this out and if it's possible?

**Comment by mashb1t** [comment]
I guess the topic is pretty hot as there were multiple similar questions popping up as of lately ^^
You can find more information here: https://github.com/lllyasviel/Fooocus/issues/1329
